cluster/images/hyperkube/master.json

```diff
@@ -23,6 +23,7 @@
               "apiserver",
               "--portal-net=10.0.0.1/24",
               "--address=127.0.0.1",
+              "--read-write-port=8080",
```

Comment:
Conversely there is no way to set up a completely insecure cluster which is a use case for some people.

This flag needs to have either the --insecure-port value or the --secure-port value so shouldn't it be a boolean `--enable-secure` type flag?